pywbem/_vendor/nocaselist/_nocaselist.py

```diff
@@ -247,7 +261,9 @@ def __mul__(self, number):
 
         Invoked using ``ncl * number``.
         """
-        if not isinstance(number, _INTEGER_TYPES):
+        # Despite using type hints, passing a non-int object does not raise
```

Comment:
Isn't this comment simply because the type hints are not processed in python but in tests tools? The comment makes it sound like a failure somewhere whereas it is a required test.

Reply:
Again, we just picked up a newer version